api/AbortController.json

```diff
@@ -0,0 +1,208 @@
+{
+  "api": {
+    "AbortController": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AbortController",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": false
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "57"
+          },
+          "firefox_android": {
+            "version_added": "57"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "ie_mobile": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "AbortController": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AbortController/AbortController",
```

Comment:
I think we should add a 'description' here: 'Constructor' it may not be obvious to everybody.

Reply:
Yes, like `"description": "<code>VRDisplayEvent()</code> constructor",` which we used in WebVR already.

Reply:
OK, description added. I didn't know about this field, so I'll add a section about it to my browser compatibility data guide